### PR TITLE
Update documentation

### LINKED ISSUE #200: Document Remote Metadata
- Set ``GALAXY_VIRTUAL_ENV`` when starting Pulsar.
- Set in Pulsar conf:
   - ``galaxy_home: /Users/john/workspace/galaxy``
   - ``galaxy_virtual_env: /Users/john/workspace/galaxy/.venv`` (probably not used)
- Set Galaxy job conf:
  -    ``<param id="remote_metadata">true</param>``
  -    ``<param id="use_metadata_binary">true</param>`` (new parameter in dev)
  -    *or*
  -    ``<param id="use_remote_datatypes">false</param>``
  -   ``<param id="remote_property_galaxy_home">/Users/john/workspace/galaxy</param>``

### PR DESCRIPTION
**Pull request by natefoo**
Documented some stuff that wasn't previously documented, removed some stuff that was outdated, did some style cleaning. Not a full sweep, but it's a start.

Fixes #200.

### CODE REVIEW

**Comment by bgruening** [comment]
@gmauro can you please give this a review if you have time?

**Comment by gmauro** [comment]
Sure

File: docs/configure.rst [hunk 1]
```diff
@@ -60,29 +84,33 @@ the following method::
 More information can be found in the `paste httpserver documentation
 <http://pythonpaste.org/modules/httpserver.html>`_.
```
Comment (gmauro) [inline docs/configure.rst:85]: This link doesn't work anymore. Should be replaced by this https://paste.readthedocs.io/en/latest/index.html

File: docs/configure.rst [hunk 2]
```diff
@@ -1,54 +1,78 @@
...
-standard sources such as `eGenix <http://www.egenix.com/products/python/pyOpenSSL/>`_.
+    $ pip install pyOpenSSL
 
 Once installed, you will need to set the option ``ssl_pem`` in ``server.ini``.
 This parameter should reference an OpenSSL certificate file for use by the
-Python paste server. This parameter can be set to ``*`` to automatically
-generate such a certificate. Such a certificate can manually be generated by
-the following method::
+Paste_ server. This parameter can be set to ``*`` to automatically generate such
+a certificate. An unsigned certificate for testing peruposes can be manually
```
Comment (gmauro) [inline docs/configure.rst:74]: Typo. purposes

**Review by gmauro — CHANGES_REQUESTED** [review]
I think **docs/files/.job_conf_sample_mq.xml.swp** can be removed.

File: docs/configure.rst (hunk 1 above)
Comment (gmauro) [reply docs/configure.rst:85]: or should be https://pythonpaste.readthedocs.io/en/latest/. I am a bit puzzled by the two websites.

Comment (natefoo) [reply docs/configure.rst:85]: pythonpaste.readthedocs.org seems to be the updated one.

Comment (natefoo) [reply docs/configure.rst]: That said, the text refers to the module documentation, which isn't being generated on RTD, so I've changed the link to the source.

**Comment by natefoo** [comment]
Thanks @gmauro!

**Pull request merged**